scipy/special/_add_newdocs.py

```diff
@@ -12988,8 +12988,41 @@ def add_newdoc(name, doc):
 
     Notes
     -----
-
+    
+    The log function used in the computation is the natural log.
```

Comment:
There should be no space between the text and the section title.
```suggestion
    The log function used in the computation is the natural log.
```